saleor/plugins/webhook/utils.py

```diff
@@ -222,3 +227,20 @@ def delivery_update(delivery: "EventDelivery", status: str):
 def clear_successful_delivery(delivery: "EventDelivery"):
     if delivery.status == EventDeliveryStatus.SUCCESS:
         delivery.delete()
+
+
+def report_event_delivery_attempt(
+    event_type: str,
+    attempt: "EventDeliveryAttempt",
+    next_retry: Optional[datetime] = None,
+):
+    if event_type not in WebhookEventAsyncType.REPORTER_EVENTS:
```

Comment:
[question] Wouldn't be more readable to explicitly specify the events here, instead of creating a specific variable in EventTypes to use it here?

Reply:
That was my initial implementation, and then I mowed it to EventTypes. So I fall back on your opinion and will move it back here 😄